### DISCUSSION TITLE
Color settings helping feature

### DISCUSSION THREAD

**Discussion by KirillArtemenko** [OnionUI/Onion#1619] [Ideas]
### Checklist

- [X] I am reporting a bug in the Onion OS - Any issues with the closed source frontend ([MainUI](https://github.com/OnionUI/MainUI-issues)), or [RetroArch](https://github.com/libretro/RetroArch) or any of the cores should be directed to their respective repositories
- [X] I have reviewed the [Onion Docs](https://onionui.github.io/docs) and did not find the answer
- [X] I have searched the existing (open and closed) issues to ensure I do not log a duplicate

### Onions OS Version

4.5.1

### Miyoo Firmware Version

202306282128

### Provide a clear and concise description of the issue

When I tried to calibrate my Miyoo Mini Plus so that its colors matched my previous console, I stuck into a loop (colibrate, reboot, test, repeat), and I would like to ask you to think about the possibility of implementing the following function. 

The calibration process is not obvious, it is clear that something is changing, but how this will affect the final result is unclear until the application is opened (after the device is rebooted). This makes the setup process quite painful.

If i could see the last screenshot during calibration, it would have been much easier to imagine the result of the applied corrections.

### Steps to Reproduce

I don’t yet know how it’s more convenient to implement this, with persistent display screenshot as a background or maybe changing the interface to screenshot by pressing a X/Y button, but I’m sure that this function will be very useful and decrease settings apply iterations count before players can reach the perfect result.

### Provide any additional context or information that may help us investigate your issue

Thank you for the great OS.

**Comment by XK9274** [comment]
Yeah there is a way this can be done... though I don't have time rn. 

If anyone gets there before me, use blue light filter script for inspiration as you can push CSC changes in via mi_disp without having to reboot the device every time. 

It wouldn't be too difficult to implement a couple of sliders (one for each value of sat/lum etc) and use R1/L1 to cycle through your screenshot collection to see how it will look in various other games/apps.

Could even be an add on to infopanel ie:

`infoPanel -csc`

To run infopanel in colour testing mode. (It already supports screenshots)
